packages/desktop/components/popups/ReceiveAddressPopup.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
     import { QR, Text, FontWeight } from 'shared/components'
-    import { AddressBox } from 'shared/components/atoms'
+    import { AddressBox } from 'shared/components'
```

Comment:
lets get these imports together 🙏🏼 we can also use the alias `@ui` for the import `shared/components` which we should be using too I think

Reply:
there are couple appearances in this PR about these 2 points